R/derive_var_trtemfl.R

```diff
@@ -72,6 +72,27 @@
 #'   *Permitted Values:* A symbol referring to a variable of the input dataset
 #'   or `NULL`
 #'
+#' @param group_var Grouping variable
+#'
+#'  If the argument is specified, it assumes that AEs are recorded as one episode
+#'  of AE with multiple lines using a grouping variable.
+#'
+#'  Events starting during treatment or before treatment and worsening afterward
+#'  are flagged. Once an AE record in a group is flagged, all subsequent records
+#'  in the treatment window are flagged regardless of severity.
+#'
+#'  *Permitted Values:* A symbol referring to a variable of the input dataset
+#'   or `NULL`
+#'
+#' @param subject_keys Variables to uniquely identify a subject.
+#'
+#'   A list of symbols created using `exprs()` is expected. This argument is only
+#'   used when `group_var` is specified.
+#'
+#'   Default: `get_admiral_option("subject_keys")[2]`
+#'
+#'   For admiral options, see [`get_admiral_option`](../reference/get_admiral_option.html)
+#'   and [`set_admiral_options`](../reference/set_admiral_options.html)
```

Comment:
This should be removed as we have a convention of not specifying the default value in the description.